pkg/sdk/testint/tasks_gen_integration_test.go

```diff
@@ -592,4 +592,30 @@ func TestInt_Tasks(t *testing.T) {
 		err := client.Tasks.Execute(ctx, executeRequest)
 		require.NoError(t, err)
 	})
+
+	t.Run("temporarily suspend root tasks", func(t *testing.T) {
```

Comment:
Okay, let's create more complicated tests please:
- more roots (started/not started - it should be possible to have only one started but let's try)
- deeper tree

I started to wonder if the current implementation of the resource is the correct one. Check the description for `RESUME | SUSPEND` in https://docs.snowflake.com/en/sql-reference/sql/alter-task#parameters: `Before resuming the root task of your Task Graph, resume all child tasks. To recursively resume the root task’s child tasks, use [SYSTEM$TASK_DEPENDENTS_ENABLE](https://docs.snowflake.com/en/sql-reference/functions/system_task_dependents_enable).`. We don't want to resume all, but we should resume some, right? Maybe let's experiment a bit.

Reply:
Please add my above comment to the follow-up ticket too.